### ISSUE TITLE
Provide `marcosnils/bin`  as an alternate installation method

### ISSUE THREAD

**Issue by joe733** [pantsbuild/scie-pants#163]
[`bin`](https://github.com/marcosnils/bin) can grab proper `scie-pants` version from `https://github.com/pantsbuild/scie-pants/releases`.

So in the `README.md` you can have:

```markdown
## Installing

### Using [`bin`](https://github.com/marcosnils/bin)

```console
$ bin i github.com/pantsbuild/scie-pants ~/.local/bin/pants
.```

### Manual

For now, you'll need to download the correct binary for your system, mark it as executable and place
it on your $PATH somewhere.

The binaries are released via [GitHub Releases](https://github.com/pantsbuild/scie-pants/releases)
for both Linux and macOS and both aarch64 and x86_64 chip architectures. Pants itself does not
support Linux aarch64 quite yet, but `scie-pants` is ready for it!

I run on Linux x86_64; so I install a stable release like so:

```console
$ curl -fLO \
  https://github.com/pantsbuild/scie-pants/releases/download/v0.1.9/scie-pants-linux-x86_64
curl -fL \
  https://github.com/pantsbuild/scie-pants/releases/download/v0.1.9/scie-pants-linux-x86_64.sha256 \
  | sha256sum -c -
chmod +x scie-pants-linux-x86_64 && mv scie-pants-linux-x86_64 ~/bin/scie-pants
.```

You can then run `scie-pants` anywhere you'd use `./pants` and other places besides. It should work
just like your existing `./pants` script but with ~33% less latency launching Pants.

If you'd like to build you own version, see the [contribution guide](CONTRIBUTING.md). There are
build instructions there.
```

---

> Preview

### Using [`bin`](https://github.com/marcosnils/bin)

```console
$ bin i github.com/pantsbuild/scie-pants ~/.local/bin/pants
```

### Manual

For now, you'll need to download the correct binary for your system, mark it as executable and place
it on your $PATH somewhere.

The binaries are released via [GitHub Releases](https://github.com/pantsbuild/scie-pants/releases)
for both Linux and macOS and both aarch64 and x86_64 chip architectures. Pants itself does not
support Linux aarch64 quite yet, but `scie-pants` is ready for it!

I run on Linux x86_64; so I install a stable release like so:

```console
$ curl -fLO \
  https://github.com/pantsbuild/scie-pants/releases/download/v0.1.9/scie-pants-linux-x86_64
curl -fL \
  https://github.com/pantsbuild/scie-pants/releases/download/v0.1.9/scie-pants-linux-x86_64.sha256 \
  | sha256sum -c -
chmod +x scie-pants-linux-x86_64 && mv scie-pants-linux-x86_64 ~/bin/scie-pants
```

You can then run `scie-pants` anywhere you'd use `./pants` and other places besides. It should work
just like your existing `./pants` script but with ~33% less latency launching Pants.

If you'd like to build you own version, see the [contribution guide](CONTRIBUTING.md). There are
build instructions there.

---

Are PRs on this welcome?

**Comment by jsirois** [comment]
I'm not sure I see the value. If you're a bin user and reading this page (prior to your edit), you know what to do (you know the GitHub url) and are not concerned about checking the hash. In effect your change doesn't help I stall scie-pants any easier (it already has a self update method via `SCIE_BOOT=update pants`), it just advertises the bin tool.

**Comment by jsirois** [comment]
Regardless, the proper place for recommendations on how to install Pants is in the pants repo / pantsbuild.org docs. I need to update this readme to point there now that this is exiting trial mode and transitioning to official mode.

**Comment by joe733** [comment]
> f you're a bin user and reading this page (prior to your edit), you know what to do (you know the GitHub url) and are not concerned about checking the hash.

Right, but `curl` does not keep track of binaries. If a beginner sees the documentation, he would prefer a simpler/shorter command. It would be even better, if the same command is capable of managing other binaries (as `bin` does in this case). I agree, the change argues more for the cause of `bin` than `scie-pants`. Yet, having an backup installation mechanism is a good thing to have.

> Regardless, the proper place for recommendations on how to install Pants is in the pants repo / pantsbuild.org docs. I need to update this readme to point there now that this is exiting trial mode and transitioning to official mode.

Ah yes, but I thought docs might be generated from `README.md`.

**Comment by sureshjoshi** [comment]
> Plus with curl, you've to be very specific on OS and architecture, whereas bin can sort that out for you.

It's worth noting that in the official Pants docs, this is the installation method:
https://www.pantsbuild.org/docs/installation

`curl --proto '=https' --tlsv1.2 -fsSL https://static.pantsbuild.org/setup/get-pants.sh`

**Comment by joe733** [comment]
> It's worth noting that in the official Pants docs, this is the installation method: https://www.pantsbuild.org/docs/installation
> 
> `curl --proto '=https' --tlsv1.2 -fsSL https://static.pantsbuild.org/setup/get-pants.sh`

Yup, that's neater, but sourcing a script directly from only isn't the best idea. Ah, but again, so is running a binary without, verifying the hash.

> Don't be put off if your PR in this case didn't quite hit home; as the proper place for Pants installation instructions are here: https://github.com/pantsbuild/pants/blob/652de6a60a1b6d27758e365f9dd51937679ef0c8/docs/markdown/Getting%20Started/getting-started/installation.md

> I don't see any direct issue with mentioning how you could get Pants setup using `bin` as an alternative to the other installation methods described there in a dedicated section towards the end of that page.

Alright :+1:
